activation/nipost.go

```diff
@@ -406,6 +406,21 @@ func (nb *NIPostBuilder) submitPoetChallenges(
 		}
 	}
 
+	misconfiguredRegistrations := make(map[string]nipost.PoETRegistration)
+	for addr := range registrationsMap {
+		if reg, ok := existingRegistrationsMap[addr]; !ok {
+			misconfiguredRegistrations[addr] = reg
+		}
+	}
```

Comment:
NIT: the map values are not used. How about using a set?
```suggestion
	misconfiguredRegistrations := make(map[string]struct{})
	for addr := range registrationsMap {
		if _, ok := existingRegistrationsMap[addr]; !ok {
			misconfiguredRegistrations[addr] = struct{}{}
		}
	}
```